apiary.apib

```diff
@@ -5,6 +5,148 @@ HOST: https://operationcode.org/
 
 API endpoints that Operation Code's Rails backend makes available to its React frontend.
 
+## Users | Create [/api/v1/users{?first_name,last_name,email,zip,password,mentor}]
```

Comment:
Can you please alphabetize the placement of these in the file.  First by model (i.e. `User`), then by action (i.e. `create`)

Reply:
Done, also adjusted `Location`